Userland/Applications/ImageViewer/ViewWidget.h

```diff
@@ -59,11 +64,12 @@ class ViewWidget final : public GUI::Frame {
     virtual void drop_event(GUI::DropEvent&) override;
 
     void set_bitmap(const Gfx::Bitmap* bitmap);
-
     void relayout();
     void reset_view();
     void animate();
 
+    Optional<size_t> get_current_index();
```

Comment:
Getters should be called `foo()`, not `get_foo()`, and they should be `const`.
If `m_current_index` is supposed to act like a cache, please mark it `mutable` and name it accordingly.

Reply:
alright, get_current_index cannot be `const` as it can update `m_files_in_same_dir.append(file);` if the current directory is empty.

Reply:
Looks like `m_files_in_same_dir` can just be marked `mutable`, it's just a lazily filled list of paths it seems.

Reply:
ok, so you mean to say I will mark `get_current_index` const but explicitly mark `m_files_in_same_dir` `mutable`. Am I right?

Reply:
Yep. and please rename this method to `current_index()` (as per the project coding style https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#right-3)